developer-docs/fixation-detector.md

```diff
@@ -6,59 +6,68 @@ page_weight = 4
 
 ### Usage {#fd-usage}
 
-#### Activation
-Any versions of the 3D fixation detector requires the 3D pupil data since it relies on the detected pupil angle for calculations. To activate 3D pupil detection, select `3D` in Capture's _General_ settings under _Detection & Mapping Mode_. In Player, it is currently not possible to generate the required 3D data from a recording that only includes 2D detected data.
+#### Online Fixation Detector
 
-In Capture, the _Fixation Detector 3D_ is loaded by default. In the future, it will be used to improve the calibration procedure.
+This plugin detects fixations based on a dispersion threshold in terms of
+degrees of visual angle with a minimal duration. It publishes the fixation
+as soon as it complies with the constraints (dispersion and duration). This
+might result in a series of overlapping fixations.
 
-In Player, the fixation detectors are not loaded by default. They are activated like every other plugin, too. See _area 2_ in [Getting Started — Player Window](#fixation-detector-dispersion-duration). Depending on the length of the recording, the Player window might freeze for a short time. This is due to the detector looking for fixations in the whole recording.
+1. Maximum Dispersion (spatial, degree): Maximal distance between all gaze locations during a fixation.
+2. Minimum Duration (temporal, milliseconds): The minimal duration in which the dispersion threshold must not be exceeded.
+3. Confidence Threshold: Data with a lower confidence than this threshold is not considered during fixation detection.
 
-#### Data access
-All fixation detectors augment the `events` object which is passed to each plugin's `update(frame,events)` method (see the [Plugin Guide](#plugin-guide)). They add a list of fixation dictionaries under the key `'fixations'`. The exact format of these fixations is described [below](#fixation-format).
+#### Offline Fixation Detector
 
-In Player, all fixations are known _a priori_ and can be referenced in all their related frames. In case of a long fixation, the detector will try to generate a single representation instead of multiple ones. In contrast, the detector in Capture will look for the shortest fixation that complies with the parameters above and add it once to `events` if it is found. The plugin will look for a new fixation afterwards. This means that a long fixation might be split into multiple fixation events. These difference in behavior is due to the different data availabilities in Capture and Player.
+This plugin detects fixations based on a dispersion threshold in terms of
+degrees of visual angle within a given duration window. It tries to maximize
+the length of classified fixations within the duration window, e.g. instead
+of creating two consecutive fixations of length 300 ms it creates a single
+fixation with length 600 ms. Fixations do not overlap.
+
+1. Maximum Dispersion (spatial, degree): Maximal distance between all gaze locations during a fixation.
```

Comment:
Maximal to `Maximum`

Reply:
I could be wrong here... https://english.stackexchange.com/questions/38260/maximum-vs-maximal - but below you use `maximum` so for consistency I would use `Maximum`